### ISSUE TITLE
File globbing bug with rake task

### ISSUE THREAD

**Issue by nicolasmccurdy** [brigade/scss-lint#836]
It seems like whenever I use file globs with the rake task, I always get a `No such file or directory @ rb_sysopen - <glob>` error, even though the linter succeeds if I run `scss-lint <glob>` or run the Rake task with the raw filenames (without globs). My current workaround is to use `Dir.glob '<glob>'` instead of `['<glob>']`, but this seems like unintended behavior.

Here's a [minimal example that fails on my machine](https://gist.github.com/nicolasmccurdy/67c79579e6f1e1673e061c416aecc3eb). I included the full Gemfile and Gemfile.lock for reproduction.

**Comment by sds** [comment]
Hey @nicolasmccurdy,

The reason you are seeing a discrepancy between running `scss-lint <glob>` and specifying that same glob in the `files` option of the Rake task is that your shell is expanding the glob pattern in the former, whereas SCSS-Lint assumes you are specifying a literal file name in the latter (since `*` is technically an allowed character in file names).

Thus the workaround you've specified is actually the correct way to use a glob pattern to match files using the `files` option of the Rake task.

Hope this helps!

**Comment by nicolasmccurdy** [comment]
Thanks for the response! I did realize that in the case os `scss-lint <glob>` the glob pattern is being handled by my shell, but I was confused because the readme seems to imply that SCSSLint::RakeTask supports globs itself:

> ## Rake Integration
...
```ruby
require 'scss_lint/rake_task'
SCSSLint::RakeTask.new do |t|
  t.config = 'custom/config.yml'
  t.args = ['--format', 'JSON', '--out', 'results.txt']
  t.files = ['app/assets', 'custom/*.scss']
end
```
...
```ruby
# Single quotes prevent shell glob expansion
rake 'scss_lint[app/assets, custom/*.scss]'
```

While the workaround we suggested does work fine for me, can we please either update the documentation to mention that globs aren't supported or implement glob support internally to make the documentation more accurate?

**Comment by sds** [comment]
Thanks for noticing this issue with the documentation, @nicolasmccurdy.

I've updated it in 77552c4. Thanks again for the report!

**Comment by nicolasmccurdy** [comment]
👍 Thanks!
